[2017-10-01 17:34:46] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:34:46] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:34:47] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:34:47] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:38:43] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:38:43] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:38:43] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:38:43] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:38:45] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:38:45] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:38:45] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:38:45] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:38:45] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:38:45] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:01] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:01] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:01] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:01] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:11] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:11] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:11] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:11] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:11] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:11] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:39:12] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:41:32] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:41:32] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:41:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:41:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:46:50] log.ERROR: ErrorException: Trying to get property of non-object in /Server/data/myweb/app/routes.php:252 Stack trace: #0 /Server/data/myweb/app/routes.php(252): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/Server/data/my...', 252, Array) #1 [internal function]: {closure}('detail', '1') #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #4 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #5 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #6 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #7 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #8 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:00] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:10] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:47:29] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:39] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:39] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:39] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:39] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:40] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:40] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:40] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:40] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:52] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:52] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:52] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:52] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:52] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:52] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:52] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:48:52] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:54:54] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:54:54] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:01] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:01] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:01] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:01] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:02] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:02] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:09] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:30] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:30] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:31] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:31] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:31] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:31] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:32] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:32] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:32] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:32] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:32] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:32] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:55:33] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:02] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:02] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:30] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:30] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:31] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:31] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:41] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:41] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:41] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:41] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:47] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:47] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:47] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:47] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:58] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:58] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:58] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 17:56:58] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2017-10-01 18:23:50] log.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException in /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /Server/data/myweb/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /Server/data/myweb/public/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
